### ISSUE TITLE
feed button optional

### ISSUE THREAD

**Issue by ainafp** [jvanz/pelican-hyde#20]
I would like to add a variable (e.g. FEED_ON) to control if the feed button is added in the sidebar (e.g. if FEED_ON then execute_lines_52_to_54 in templates/sidebar.html). Not sure if there is already a way to do this?

Thanks a lot! I love the template!

**Comment by jvanz** [comment]
No, as far as I can remember, this is not possible. If you can add this feature, please go ahead. I have issue to find time to work in the template this days. It's not my top priority.  But you want to add this, please, open a PR! I'll love review it! :)
